modules/BookManager.js

```diff
@@ -0,0 +1,23 @@
+import bookCollection from './BookCollection.js';
+
+function displayBooks() {
```

Comment:
- [ ] As using ES6 syntax is a requirement in this milestone, kindly consider using ES6 arrow functions instead of random function keyword.